### ISSUE TITLE
Do not prompt for Update again when Cancelled

### ISSUE THREAD

**Issue by deadcoder0904** [buttercup/buttercup-desktop#618]
New update has prompted which has support for some language which is not useful for me. So when I click on **Not Now** when prompted for **Update** then it must not prompt it again every time I open the app. 

Also, another feature request must be a way to Skip All Updates.

**Comment by perry-mitchell** [comment]
Hi @deadcoder0904 - I'm not sure if we exactly want to support skipping versions. Sure, single-language updates can be frustrating when they're not relevant, but many applications **force** updates and just prompt for a restart. Often times patch updates will be for security features, and these we do not want people to skip - by using our software our brand and reputation are on the line when it comes to security, so ideally everyone would always use the most up-to-date version of Buttercup without exception.

**Comment by deadcoder0904** [comment]
Yeah exactly this update was the exception but its irritating that it always prompts up for updates. A **Skip Update** option is better IMHO. 

At least as a user, I think its much better because I open ButterCup at least 10 times a day. 

From a dev perspective, yeah security updates are nice. Also, I like auto-updates like VSCode does but I understand you don't want to go that way :)

I love ButterCup. Its just amazing ❤️

**Comment by perry-mitchell** [comment]
Thank you so much! That’s really awesome to hear. We appreciate the feedback.

Perhaps some option for devs to publish updates that can be skipped and updates that cannot? These language patches could easily be skipped for some whilst security updates would be mandatory.

There might be some nicer way to prompt, as well. For instance we could prompt only for important updates, and the other less important ones could simply bring up an indicator in the UI. I’ll speak about it with my colleague..

**Comment by deadcoder0904** [comment]
Yeah that sounds pretty cool. Even if you guys abandon maintaining it, I'll still use it. Its reached at a point where everything works pretty fine for me 😂

Thank you for making it :)
